queries.sql

```diff
@@ -48,3 +48,48 @@ BEGIN TRANSACTION;
 UPDATE animals SET weight_kg = 20.4 WHERE name = 'Boarmon';
 SELECT * FROM animals;
 COMMIT;
+
+ALTER TABLE animals DROP COLUMN species;
+ALTER TABLE animals ADD COLUMN species_id INT;
+ALTER TABLE animals ADD COLUMN owner_id INT;
+ALTER TABLE animals ADD CONSTRAINT fk_species FOREIGN KEY (species_id) REFERENCES species(id);
+ALTER TABLE animals ADD CONSTRAINT fk_owner FOREIGN KEY (owner_id) REFERENCES owners(id);
+\d animals;
+SELECT * FROM owners;
+SELECT* FROM species;
+UPDATE animals SET species_id = CASE WHEN name LIKE '%mon' THEN 2 ELSE 1 END;
+SELECT * FROM animals;
+UPDATE animals
+SET owner_id = (
+  SELECT o.id
+  FROM owners o
+  WHERE o.full_name = CASE
+    WHEN animals.name = 'Agumon' THEN 'Sam Smith'
+    WHEN animals.name IN ('Gabumon', 'Pikachu') THEN 'Jennifer Orwell'
+    WHEN animals.name IN ('Devimon', 'Plantmon') THEN 'Bob'
+    WHEN animals.name IN ('Charmander', 'Squirtle', 'Blossom') THEN 'Melody Pond'
+    WHEN animals.name IN ('Angemon', 'Boarmon') THEN 'Dean Winchester'
+    ELSE NULL
+  END
```

Comment:
- [ ] Kindly make these changes in the `data.sql file. I noted it's missing there, but kindly transfer these changes there. Its best practice for separating all three files with their functions 👍

Reply:
-  Moved files from `queries.sql` file to `data.sql` file to resolve bug.

Reply:
This change has been done well